packages/apps/dev-wallet/src/utils/id-to-color.ts

```diff
@@ -0,0 +1,15 @@
+export function idToColor(id: string) {
```

Comment:
good for now but we might refine this approach later for example maybe designers prefer to only use a limited list of colors instead of creating a random one